content/docs/ops/create-org.md

```diff
@@ -0,0 +1,31 @@
+---
+menu:
+  docs:
+    parent: tenants
+title: Creating organizations
+---
+
+## Sandboxes
+
+New sandbox organizations are automatically created when users sign in for the first time, via the [sandbox bot](https://github.com/18F/cg-sandbox-bot).
+
+## Non-sandboxes
+
+Non-sandbox organizations need to be created by hand. To do so:
+
+1. The [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1) should be filled out by you or the tenant (if it hasn't been already).
```

Comment:
I don't think you need the parenthetical comment here. If someone filled it out, they know they filled it out. If they didn't, they're not going to know just by looking at that form. If they should ask someone else first we should tell them that; otherwise let's just not say anything and accept there will be some duplicates.